README.md

```diff
@@ -72,6 +72,10 @@ Example:
 
     alias: `poweroff`
 
+  - `env:switch`: A project can have both environments configured: lando and ddev, so the "switch" command allows you to change the default environment.
+
+    alias: `switch`
```

Comment:
PLease add all the command aliases, also I think  we are missing something into this description, I was thinking something more inline with : This Command helps you "hot  swap" between local dev enviroments (Lando and DDev).

Something like that , edit as you need

Reply:
Hi @rigoucr done!